Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+                       
+            string firstName = returnString("Enter First Name");
```

Comment:
Second person missing

Reply:
I actually thought that we just do it twice to make sure the information is correct. 
i will add the 2nd person in.